test.py

```diff
@@ -75,11 +75,16 @@ def run_single_test(config, tutorial, temp_file):
         sys.exit(1)
 
     tasks = open(os.path.join(tute_dir, ".tasks"), 'r').read()
+    shutil.rmtree(build_dir)
```

Comment:
I think this will fail if `build_dir` doesn't exist. To make it properly robust against both cases, the following should work:

```suggestion
    shutil.rmtree(build_dir, ignore_errors=True)
```